packages/react-sdk/src/state/communication/discovery/matrixRtcSessionManagerImpl.ts

```diff
@@ -45,16 +51,30 @@ export class MatrixRtcSessionManagerImpl implements SessionManager {
   private readonly leaveSubject = new Subject<void>();
   private readonly sessionJoinedSubject = new Subject<Session>();
   private readonly sessionLeftSubject = new Subject<Session>();
-  // This subject is used to track the state of all sessions, allowing us to
-  // detect sessions and manage their lifecycle, including adding expiration dates.
+  private readonly activeFocusSubject = new ReplaySubject<RTCFocus>(1);
   private readonly sessionSubject = new Subject<SessionState>();
   private sessions: StateEvent<RTCSessionEventContent>[] = [];
   private joinState: { whiteboardId: string; sessionId: string } | undefined;
+  private memberFocus: RTCFocus | undefined = undefined;
```

Comment:
We don't work yet with `memberFocus` in this PR, I would prefer to pass `undefined` explicitly to `makeFociPreferred` (maybe with TODO) to make sure this field is not really used somewhere else.